src/style.css

```
@@ -0,0 +1,177 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+/* #region Normalize */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+
+.cards {
+  display: flex;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 404px;
+  width: 200px;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  flex-wrap: wrap;
+}
```

```suggestion
.card { 
  width: 200px;
  border-radius: 5px;
  border: 1px solid #f3f3f3;
}
```
White color for background is default. Why here flex, block elements by default become one under another. Add padding to card from mock-up; top - 32 r,b,l - 16